README.md

```diff
@@ -65,42 +63,59 @@ List of dependencies:
 6. gcc (>=5.4.0)
 7. faiss-gpu (>=1.4.0) - To install with conda: ```conda install -c pytorch faiss-gpu cuda92```
 
-### Setup steps
+### Installing from Source:
 
-To clone:
+Once dependencies are present, follow the steps below:
 
+1. Clone the repository.
+```bash
+$ git clone --recurse-submodules https://github.com/rapidsai/cuml.git
 ```
-git clone --recurse-submodules https://github.com/rapidsai/cuml.git
+
+2. Build and install `libcuml` (the C++/CUDA library containing the cuML algorithms), starting from the repository root folder:
+```bash
+$ cd cuML
+$ mkdir build
+$ cd build
+$ cmake ..
 ```
 
-To build the python package, in the repository root folder:
+Note: if using a conda environment (recommended currently), then cmake cen be configured appropiately via:
 
-```
-python setup.py install
+```bash
+$ cmake .. -DCMAKE_INSTALL_PREFIX=$CONDA_PREFIX
 ```
 
-### Building CuML:
+Build `libcuml`:
 
-### Running tests
+```bash
+$ make -j
+$ make install
+```
 
-To test the C++ algorithms using googletests, in the repository root folder:
+To run tests (optional):
 
 ```bash
-$ cd cuML
-$ mkdir build
-$ cd build
-$ cmake ..
-$ make -j 
-$ # Instead of previous step if you like you can use cmake directly
-$ cmake --build . --config Release -- -j
 $ ./ml_test
 ```
 
-To list available test:
```

Comment:
I think command description which shows available tests should be persist. It's usefull information for person (even it's simple one, but it's one of the usefull command. Other are: `--gtest_filter`,  `--gtest_shuffle` ` --gtest_repeat=10`) `./ml_test --gtest_list_tests`

Reply:
I agree, I'll add them back :)